OctopusDSC/DSCResources/cOctopusServer/cOctopusServer.psm1

```diff
@@ -237,7 +262,18 @@ function Import-ServerConfig {
 
     if (Test-OctopusVersionSupportsShowConfiguration) {
         $rawConfig = & $octopusServerExePath show-configuration --format=json-hierarchical --noconsolelogging --console --instance $InstanceName
-        $config = $rawConfig | ConvertFrom-Json
+
+        # handle a specific error where an exception in registry migration finds its way into the json-hierarchical output
```

Comment:
Can you give specific details about what the error was?

Reply:
I'll add a reference back to the Issue #179